tests/fortex/health/processors/temporal_mention_tagging_processor_test.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2022 The Forte Authors. All Rights Reserved.
```

Comment:
I suggest we check one more thing. I was under the impression that we would be using Normalizer in conjunction with the temporal tagger. I think we still support that, if we replace `entry_type` in Normalizer config, from `Document` to `TemporalTag`, but would be great if we can also have a test for that. Essentially, we add both the processors, `TemporalMentionTaggingProcessor` and `TemporalMentionNormalizingProcessor`, and check what output the pipeline produces and if it's as expected.